docs/docs/reference/other-new-features/indentation-experimental.md

```diff
@@ -18,6 +18,10 @@ This variant is more contentious and less stable than the rest of the significan
 
 Similar to what is done for classes and objects, a `:` that follows a function reference at the end of a line means braces can be omitted for function arguments. Example:
 ```scala
+//{
+import language.experimental.fewerBraces
+def times(i: Int)(fn: => Unit) = (0 to i).foreach(x => fn)
```

Comment:
Should we show both the import and the def, in the snippet, instead of putting them to the snippet context?